nixos/doc/manual/development/settings-options.section.md

```diff
@@ -60,6 +60,22 @@ have a predefined type and string generator already declared under
 
 `pkgs.formats.ini` { *`listsAsDuplicateKeys`* ? false, *`listToValue`* ? null, \.\.\. }
 
+:   A function taking an attribute set with values
+
+    `listsAsDuplicateKeys`
+
+    :   A boolean for controlling whether list values can be used to
+        represent duplicate INI keys
+
+    `listToValue`
+
+    :   A function for turning a list of values into a single value.
+
+    It returns a set with INI-specific attributes `type` and `generate`
+    as specified [below](#pkgs-formats-result).
+
+`pkgs.formats.iniWithGlobalSection` { *`listsAsDuplicateKeys`* ? false, *`listToValue`* ? null, \.\.\. }
```

Comment:
I think this section should have a bit more information, especially the fact that a submodule with separate options for the global and other sections is returned. A warning about it not being compatible with `ini` and how it compares to it would also be great.